api/kroxylicious-filter-api/src/main/java/io/kroxylicious/proxy/filter/FilterInvokers.java

```diff
@@ -17,6 +17,10 @@
  */
 public class FilterInvokers {
```

Comment:
Is it me, or is the indentation of the inner records (e.g. `RequestResponseInvoker`) of this class incorrect?

Reply:
yeah, Sam commented on this earlier (can't seem to locate that comment). This is the output of the maven plugin so I've gone with it for now with the intent to fix the auto formatting up later.